nuxeo-platform-spreadsheet-web/src/main/js/app/ui/spreadsheet.js

```diff
@@ -24,10 +25,9 @@ import {Query} from '../nuxeo/rpc/query';
  */
 class Spreadsheet {
 
-  constructor(container, connection, layout, columns, pageProvider, language) {
+  constructor(container, connection, resultLayoutName, resultColumns, pageProviderName, language) {
```

Comment:
why the change of names? these are just longer and in this context there's no mix up, we have one layout, one set of columns and one pageProvider....